devices/usbc/client.h

```diff
@@ -0,0 +1,179 @@
+/*
+ * Phoenix-RTOS
+ *
+ * phoenix-rtos-loader
+ *
+ * usb client
+ *
+ * Copyright 2019-2021 Phoenix Systems
+ * Author: Kamil Amanowicz, Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef _CLIENT_H_
+#define _CLIENT_H_
+
+#include "../types.h"
+
+#include "usbclient.h"
+
+
+#define USB_BUFFER_SIZE 0x1000
+
+#define ENDPOINTS_NUMBER 7
+#define ENDPOINTS_DIR_NB 2
+
+#define MIN(X, Y)           (((X) < (Y)) ? (X) : (Y))
+
+#define DTD_SIZE(dtd)       ((dtd->dtd_token >> 16) & 0x7fff)
+#define DTD_ERROR(dtd)      (dtd->dtd_token & (0xd << 3))
+#define DTD_ACTIVE(dtd)     (dtd->dtd_token & (1 << 7))
+
+
+/* device controller structures */
+
+/* data transfer descriptor */
+typedef struct _dtd_t {
+    u32 dtd_next;
+    u32 dtd_token;
```

Comment:
In the files in directory `devices/usbc:
```
client.h
usbclient.h
usb.h
```
correct indentation.

Reply:
Done